src/components/ProductImageUpload.tsx

```
@@ -91,6 +124,52 @@ export function ProductImageUpload({
+        
+        // 2. 포인트 차감
+        const usePointsResponse = await pointApiService.usePoints({
+          amount: requiredPoints,
+          refererId: parseInt(selectedModel.id), // 모델 ID를 참조 ID로 사용
+          refererType: 'ORDER'
+        });
+
+        console.log('usePointsResponse:', usePointsResponse);
+
+        console.log('포인트 차감 성공!');
+        toast.success(`${requiredPoints.toLocaleString()} 포인트가 차감되었습니다. 남은 포인트: ${usePointsResponse.balanceAfter.toLocaleString()}`);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)